jsonrpsee/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "jsonrpsee"
+description = "JSON-RPC crate"
+version = "0.2.0-alpha.4"
+authors = ["Parity Technologies <admin@parity.io>", "Pierre Krieger <pierre.krieger1708@gmail.com>"]
+license = "MIT"
+edition = "2018"
+
+[dependencies]
+http-client = { path = "../http-client", version = "0.2.0-alpha.4", package = "jsonrpsee-http-client", optional = true }
+http-server = { path = "../http-server", version = "0.2.0-alpha.4", package = "jsonrpsee-http-server", optional = true }
+ws-client = { path = "../ws-client", version = "0.2.0-alpha.4", package = "jsonrpsee-ws-client", optional = true }
+ws-server = { path = "../ws-server", version = "0.2.0-alpha.4", package = "jsonrpsee-ws-server", optional = true }
+types = { path = "../types", version = "0.2.0-alpha.4", package = "jsonrpsee-types", optional = true }
```

Comment:
we could remove `types` and re-export those from the clients instead.

Such as `traits` and the `types` used in the normal API/traits.

Reply:
I agree, let's do that.